### PR TITLE
chain: Reduce number of module parameters by hardcoding sizing numbers

### PR DESCRIPTION
**Pull request by sorawit**

### CODE REVIEW

File: chain/x/oracle/keeper/raw_request.go [hunk 1]
```diff
@@ -19,9 +19,9 @@ func (k Keeper) SetRawRequest(ctx sdk.Context, rid types.RID, data types.RawRequ
 
 // AddRawRequest performs all sanity checks and adds a new raw request to the store.
 func (k Keeper) AddRawRequest(ctx sdk.Context, rid types.RID, data types.RawRequest) error {
-	if err := k.EnsureLength(ctx, types.KeyMaxCalldataSize, len(data.Calldata)); err != nil {
-		return err
-	}
+	// if err := k.EnsureLength(ctx, types.KeyMaxCalldataSize, len(data.Calldata)); err != nil {
```
Comment (taobun) [inline chain/x/oracle/keeper/raw_request.go:23]: Comment or deleted?

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/x/oracle/keeper/raw_request.go (hunk 1 above)
Comment (sorawit) [reply chain/x/oracle/keeper/raw_request.go:23]: Added comment!

**Review by Benzbeeb — DISMISSED** [review]

File: chain/x/oracle/types/limits.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package types
+
+// nolint
+const (
+	MaxNameLength        = 128
+	MaxDescriptionLength = 4096
+	MaxClientIDLength    = 128
+	MaxSchemaLength      = 128
```
Comment (taobun) [inline chain/x/oracle/types/limits.go]: 128 is too low 500 I think enough

File: chain/x/oracle/types/params.go [hunk 3]
```diff
@@ -35,10 +23,6 @@ const (
 	// Default value is set 1 kB.
 	DefaultMaxResultSize = uint64(1 * 1024)
 
-	// The maximum size of name length.
-	// Default value is 280
-	DefaultMaxNameLength = uint64(280)
-
 	// The maximum size of description length.
 	// Default value 4096
 	DefaultDescriptionLength = uint64(4096)
```
Comment (taobun) [inline chain/x/oracle/types/params.go]: Delete this

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — DISMISSED** [review]

**Pull request merged**